### PR TITLE
CORDA-2763 - Log censored config at startup

### PR DESCRIPTION
**Pull request by tlil**
JIRA ticket: [CORDA-2763](https://r3-cev.atlassian.net/browse/CORDA-2763)

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/internal/subcommands/ValidateConfigurationCli.kt [hunk 1]
```diff
@@ -26,7 +26,7 @@ internal class ValidateConfigurationCli : CliWrapperBase("validate-configuration
             return "for path: \"$pathAsString\": $message"
         }
 
-        internal fun logRawConfig(config: Config) = logger.debug("Actual configuration:\n${V1NodeConfigurationSpec.describe(config, Any?::toConfigValue).render(configRenderingOptions)}")
+        internal fun logRawConfig(config: Config) = logger.info("Actual configuration:\n${V1NodeConfigurationSpec.describe(config, Any?::toConfigValue).render(configRenderingOptions)}")
```
Comment (anthonykeenan) [inline node/src/main/kotlin/net/corda/node/internal/subcommands/ValidateConfigurationCli.kt]: If it's just a case of changing the logging level, I'm inclined to ask Matthew whether he's alright with that and not make any changes. It does put a lot of junk in the log files. WDYT?

Comment (tlil) [reply node/src/main/kotlin/net/corda/node/internal/subcommands/ValidateConfigurationCli.kt]: Well, if you switch on debug-level logging, you get a very verbose node log. :) Also, note that we're not dumping the fully annotated config here. Just the provided one with censored fields. Personally, I don't see a problem with 20-80 lines of config dumped in a 1000s-line long log. And to his point previously, it's in production that this is useful for validation during support requests, etc., and most people won't run a production node with debug-level logging, I don't think.

Comment (mnesbit) [reply node/src/main/kotlin/net/corda/node/internal/subcommands/ValidateConfigurationCli.kt]: Customers run at info. We always get sent a garbled story and a full log encapsulating the true config is key to problem resolution. The config must hide any passwords (and probably the DB jdbc URL for that reason too)

Comment (anthonykeenan) [reply node/src/main/kotlin/net/corda/node/internal/subcommands/ValidateConfigurationCli.kt]: Ok sounds sensible, in that case looks fine to me.

Comment (tlil) [reply node/src/main/kotlin/net/corda/node/internal/subcommands/ValidateConfigurationCli.kt]: > Customers run at info. We always get sent a garbled story and a full log encapsulating the true config is key to problem resolution. The config must hide any passwords (and probably the DB jdbc URL for that reason too)

@mnesbit, yep, this is exactly what it does :-)

**Pull request merged**